packages/client/.eslintignore

```diff
@@ -1,2 +1,3 @@
 __generated__/*
 serviceWorker/*
+AddFeatureFlagMutation.ts
```

Comment:
AddFeatureFlagMutation was being used for the Spotlight feature. We wanted users to be able to enter a cheat code to add themselves to the feature. 

This is no longer in use but will be a helpful feature in the future. To prevent the `Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser` eslint complaint, I've added it to the eslintignore file here.

Reply:
-1 ignoring the file completely is not good since it will not be linted. We should rather add this explicitly to the project so it's linted correctly. You should add the file here https://github.com/ParabolInc/parabol/blob/066796ca63a54773fab9363e7f5d28eee8d0202c/packages/client/tsconfig.eslint.json instead

Reply:
Makes sense, thanks. Updated!